documentation/contributing/releases/index.markdown

```diff
@@ -0,0 +1,135 @@
+---
+author: tylerjw
+layout: page
+slug: releases
+title: Releases
+redirect_from: "/documentation/contributing/releases.html"
+---
+
+# Patch Releases
+
+MoveIt has a 6 week cadence for patch releases.  This pace is a balance between releasing new improvements quickly and preserving the stability of releases.  We are choosing this schedule to reduce the amount of work per release, reduce the time it takes to get an improvement released and make it easier to schedule for the Picknik team.  These releases will include the main MoveIt repo along with any dependencies we control if they have changed (ie. moveit_msgs).
+
+## Upcoming releases
+
+* **Noetic (1.1.1) and Foxy (2.0.1)**
+  * Oct 02, 2020 - Code freeze
```

Comment:
Perhaps a more efficient way of tracking these dates is via a dedicated GCal that has recurring events for all three of these dates, every 6 weeks. Advantages:

- Can embed that calendar here
- Can add it to your calendar, and anyone else who is curious (Robert), so this doesn't get forgotten
- Will auto-make dates to infinity...

Reply:
We will likely have to deviate from this at some point anyway, so I don't like documenting it here in markdown.

Reply:
Moved this into a google calendar.  That will make it easier for us to change in the future and if we don't have to change it we won't have to do it via a pull request here.